siem_integrations/clx_query/bin/clx_query.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
```

Comment:
Needs standard copyright at top of the file.

Reply:
Added copyright information.